shared/components/Animation/__tests__/Reveal.spec.jsx

```diff
@@ -24,3 +24,25 @@ describe('Reveal', () => {
     expect(reveal).toMatchSnapshot()
   })
 })
+
+describe('accessibility', () => {
+  it('marks the panel as an expandable trigger for content for screen readers', () => {
+    const reveal = shallow(
+      <Reveal timeout={500} in={true} height={300}>
+        {() => <div>Content to reveal</div>}
+      </Reveal>
+    )
+
+    expect(reveal.dive()).toHaveProp('aria-hidden', false)
```

Comment:
Doing `reveal.dive()` is a pattern we used to do a lot, but have starting moving away from because it is pretty brittle and relies on an exact HTML structure, so its likely to break when you don't want it to.

Instead, try placing a `data-testid="some id for your tests"` on the appropriate element in the Reveal component and then use a selector to target it in your test. We also will create a `doShallow` or `doMount` function to encapsulate rendering. This is especially helpful to hide away props that don't matter for the behaviour you are testing. In this case you set `timeout` and `height` because they are required, but those don't matter for this test.

Example: https://github.com/telusdigital/tds-core/blob/master/packages/Tooltip/__tests__/Tooltip.spec.jsx#L16-L27

Let me know if you have questions about this stuff. It is probably easier to show than describe in a comment.